securityc/README.md

```diff
@@ -0,0 +1,162 @@
+# securityc
```

Comment:
Should we add that architecture image to the README?

Reply:
Might be useful also to briefly say when this would useful, e.g. for development. Should also include some mention of this in the primary read me too with a link to this read me